docs/templating/markdown.html

```diff
@@ -147,16 +178,26 @@ <h2 class="heading-2"><span class="blue-text bold">Step 2:</span> Create the tem
           <pre><code>swift package generate-xcodeproj</code></pre>
           <p>This is to enable Xcode to detect the new directory.</p>
         </div>
-        <div class="underline"></div>
-        <h2 class="heading-2"><span class="blue-text bold">Step 3:</span> Populate the Markdown template file</h2>
-        <p>First we need some Markdown to render.</p>
-        <p>So in the <strong>book.md</strong> file we will add:</p>
+        <p>Now we can add some Markdown to render:</p>
+        <p>So in the <strong>book.md</strong> file add:</p>
         <pre><code># Books Header
 
 Description with **bold** text
 
 * Book1
 * Book2</code></pre>
+        <div class="underline"></div>
+        <h2 class="heading-2"><span class="blue-text bold">Step 4:</span> Render the Markdown</h2>
+        <p>We've just added the KituraMarkdown template engine to our router, so now we can use it in a route.</p>
```

Comment:
"We've just added" doesn't make sense now this text has moved so far from its original location!